README.md

```diff
@@ -34,15 +34,18 @@ docker pull rabbitmq:3-management
 Start Rabbit and Postgres
 
 ```bash
-docker run -d --name postgres -e POSTGRES_PASSWORD=testing -p 5432:5432 -e POSTGRES_USER=testing postgres:9.6-alpine
+docker run -d --hostname postgres --name postgres -e POSTGRES_PASSWORD=testing -p 5432:5432 -e POSTGRES_USER=testing postgres:9.6-alpine
```

Comment:
Is the hostname thing necessary? Isn't it enough the name here and link at the other side?